modules/ROOT/pages/bdm-management-in-bonita-bpm-portal.adoc

```diff
@@ -83,7 +83,6 @@ This limitation is well known and will be addressed in a future Bonita version.
 For Enterprise, Performance, and Efficiency editions only.
 ====
 +
```

Comment:
Nothing to do with this PR, but there are many `+` characters on this page that shouldn't be there.

Reply:
See page rendering https://bonitasoft-bonita-doc-build_preview-pr-1491.surge.sh/bonita/2021.1/bdm-management-in-bonita-bpm-portal

Reply:
good catch, the issue is only in 2021.1, I guess this is because in this version, we have introduced a dedicated paragraph for `Manage BDM Access Control` 
There was an issue with the numbered list, and after its fix in 7.9, the doc update propagation has added this extra `+` in `2021.1`.
@educhastenier  Can you create a new PR to fix it?


### Display in 2021.1

![image](https://user-images.githubusercontent.com/27200110/112841084-5b8bfc80-90a0-11eb-89c6-a04ddd3e0ff5.png)


### Display in 7.9-7.11

![image](https://user-images.githubusercontent.com/27200110/112841000-44e5a580-90a0-11eb-8b08-4fcda33b861c.png)

Reply:
yes, I will

Reply:
managed by #1515

Reply:
Ok, this is the PR: https://github.com/bonitasoft/bonita-doc/pull/1515